### PR TITLE
[BRD-125] - Monorepo

### PR DESCRIPTION
**Pull request by ilteoood**
Monorepo introduction for packages:
- be-config
- core
- fe-container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,26 +26,55 @@ the correct version if you don’t already have it.
...
-yarn run coverage
+yarn coverage
+```
+
+These two commands, will install the dependencies and run the tests with the coverage report that you can view as an
+HTML page in `packages/*/coverage/lcov-report/index.html`.
+
+### Run a package script
+
+This folder use yarn as package manager.
```
Comment (epessina) [inline README.md:40]:
```suggestion
This folder uses yarn as package manager.
```

File: README.md [hunk 2]
```diff
@@ -24,26 +26,55 @@ the correct version if you don’t already have it.
...
+
+To run a script in a sub package, you should use `yarn workspace PACKAGE_NAME SCRIPT_NAME`.
```
Comment (epessina) [inline README.md:42]:
```suggestion
To run a script in a sub package, you can run `yarn workspace PACKAGE_NAME SCRIPT_NAME`.
```

File: package.json [hunk 3]
```diff
@@ -2,82 +2,27 @@
...
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "start": "npm-run-all 'fe-container start' 'be-config start' 'core start'",
+    "lerna": "lerna",
+    "be-config": "yarn workspace @mia-platform-microlc/fe-website",
```
Comment (davidebianchi) [inline package.json:12]: The scope should be `@mia-platform`, not `@mia-platform-microlc`

**Review by davidebianchi — CHANGES_REQUESTED** [review]
Pipelines fails because by default with monorepo the added modules are in package, and to avoid error you should add an option (`-W`) to yarn add command.

Coveralls could be also added as dev dependency of the root to avoid this issue, what do you think?

The pipeline should be changed to reflect the new structure

**Comment by ilteoood** [comment]
Hi @davidebianchi ,
I opted for the inclusion of `coverall` and `handlebars@4.5.0` in project.
I've also changed some steps in the pipeline.

**Pull request merged**